main.js

```diff
@@ -0,0 +1,121 @@
+// make click button work
+// set up API and test that it works
+// send info from callback to a data array
+// info is pushed from data array to render
+// weather should be replaced with new data if new locaton is searched
+// create 5 day forecast and render it
+
+var dayOfWeek = function(txt) {
+  let date = new Date(txt);
```

Comment:
stay consistent with your variable declarations var is pre-es6 where as let/const are considered newer as of es6